admin-dev/themes/new-theme/js/theme.js

```diff
@@ -47,6 +42,13 @@ import './product-page/index';
 import './translation-page/index';
 
 import Header from './header.js';
+import initDatePickers from './app/utils/datepicker';
+
+const $ = global.$;
```

Comment:
You may need to use `window` instead of `global`

Reply:
Should I need to replace every occurrences  of `global` in other files?